WebDriverAgentLib/Utilities/FBXCAXClientProxy.m

```diff
@@ -74,7 +74,12 @@ - (BOOL)hasProcessTracker
   static BOOL hasTracker;
   static dispatch_once_t onceToken;
   dispatch_once(&onceToken, ^{
-    hasTracker = [FBAXClient valueForKey:@"applicationProcessTracker"] != nil;
+    @try {
+      hasTracker = [FBAXClient valueForKey:@"applicationProcessTracker"] != nil;
```

Comment:
can you please replace valueForKey with respondsToSelector?

Reply:
let me try 🆗

Reply:
worked 👍